src/controllers/auth.controllers.js

```diff
@@ -1,128 +1,152 @@
+/* eslint-disable arrow-parens */
 import { validationResult } from 'express-validator';
 import bcrypt from 'bcryptjs';
 import { generateToken } from '../utils/authentication';
 import User from '../models/user.model';
 import Error from '../middleware/error/ErrorHandler';
 
+// signup endpoint
```

Comment:
good that u split these validators but these are not api controllers so you should move these to somewhere and reuse it